digiwf-apps/packages/apps/digiwf-tasklist/src/views/GroupTaskDetail.vue

```diff
@@ -114,74 +117,53 @@ export default class MyTaskDetail extends Vue {
+    return {id: this.id, type: "task"}
+  };
 
   @Provide('apiEndpoint')
-  apiEndpoint = EngineServiceApiConfig.base;
+  apiEndpoint = ApiConfig.base;
 
   created() {
-    this.loadTask();
+    this.isLoading = true
+    loadTask(this.id)
+      .then(result => {
+        this.isLoading = false;
+        if (result.data) {
+          this.task = result.data.task
+          this.errorMessage = "";
+        }
+        if (result.error) {
+          this.errorMessage = result.error;
+        }
+      })
   }
 
   async checkTaskAssignment(): Promise<void> {
-    await this.loadTask();
-    if (this.task?.assignee) {
-      const currentUser: UserTO = this.$store.getters['user/info'];
-      if (this.task?.assignee != currentUser.lhmObjectId){
-        this.showModal = true;
-        setTimeout(() => this.showModal = false, 10000);
-      }
-      else {
-        router.push({path: '/task/' + this.id});
-      }
-    }
-    else {
-      this.assignTask();
-    }
-  }
-
-  async assignTask(): Promise<void> {
-    this.showModal = false;
-    try {
-
-      const cfg = EngineServiceApiConfig.getAxiosConfig(FetchUtils.getPOSTConfig({}));
-      await HumanTaskRestControllerApiFactory(cfg).assignTask(this.id);
-
-      this.$store.dispatch('tasks/getTasks', true);
-      this.$store.dispatch('openGroupTasks/getTasks', true);
-      this.$store.dispatch('assignedGroupTasks/getTasks', true);
-      this.errorMessage = "";
-      router.push({path: '/task/' + this.id});
-    } catch (error) {
-      this.errorMessage = 'Die Aufgabe konnte nicht zugewiesen werden.';
-    }
-  }
-
-  async loadTask(): Promise<void> {
-    const loadingTimeout = setTimeout(() => this.isLoading = true, 500);
-    try {
-      const cfg = EngineServiceApiConfig.getAxiosConfig(FetchUtils.getGETConfig());
-      cfg.baseOptions.validateStatus = function (status: number) {
-        return status >= 200 && status < 500;
-      }; // override axios default impl. (holding back http statuses >= 300)
-      const res = await HumanTaskRestControllerApiFactory(cfg).getTaskDetail(this.id);
-      if (res.status >= 200 && res.status < 300) { // as in axios default impl.
-        this.task = res.data;
-        this.errorMessage = "";
-      } else {
-        if (res.status === 404) {
-          this.errorMessage = 'Die Aufgabe oder der zugehörige Vorgang wurden bereits abgeschlossen. Die Aufgabe kann daher nicht mehr angezeigt oder bearbeitet werden.';
+    loadTask(this.id) // FIXME: why should the task be loaded again?
+      .then(result => {
+        if (result.data?.task?.assigneeId) {
+          const currentUser: UserTO = this.$store.getters['user/info'];
+          if (this.task?.assigneeId != currentUser.lhmObjectId) {
+            this.showModal = true;
+            setTimeout(() => this.showModal = false, 10000); // FIXME why do we use auto close?
+          } else {
+            router.push({path: '/task/' + this.id}); // FIXME: why auto forwarding to
```

Comment:
Ich schätze parallele Bearbeitung des Tasks in mehreren Tabs. Wenn ich mir das zuweise in einem anderen Tab und bei dem ersten noch in der Gruppenansicht bin. Kann so bleiben, denke ich